code/modules/religion/sects/shadow_sect.dm

```diff
@@ -351,4 +390,465 @@
+		sect.obelisk_number = sect.obelisk_number + 1
+		obelisk.AddComponent(/datum/component/dark_favor, user)
+		obelisk.anchored = anchored
+		if(anchored)
+			sect.active_obelisks += obelisk
+			sect.active_obelisk_number += 1
+			obelisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
+		Destroy()
+	if(sect.grand_ritual_level == 3)
+		var/obj/structure/destructible/religion/shadow_obelisk/after_rit_1/after_rit_2/after_rit_3/obelisk = new(our_turf)
+		sect.obelisks += obelisk
+		sect.obelisk_number = sect.obelisk_number + 1
+		obelisk.AddComponent(/datum/component/dark_favor, user)
+		obelisk.anchored = anchored
+		if(anchored)
+			sect.active_obelisks += obelisk
+			sect.active_obelisk_number += 1
+			obelisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
+		Destroy()
+
+// Grand rituals themselves
+
+/datum/religion_rites/grand_ritual_one
+	name = "Grand ritual: Beconing shadows"
+	desc = "Convince shadows to take interest in your cult. They will carry information between their kind and their mere presence will make the darkness holier."
+	ritual_length = 35 SECONDS
+	ritual_invocations = list(
+		"Shadows hear me...",
+		"... Come to your kin ...",
+		"... Help us spread darkness ...")
+	invoke_msg = "I summon you to our beacons!"
+
+/datum/religion_rites/grand_ritual_one/perform_rite(mob/living/user, atom/religious_tool)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	if(sect.grand_ritual_level > 0)
+		to_chat(user, span_warning("You already performed this ritual!."))
+		return FALSE
+	if(!isshadow(user))
+		to_chat(user, span_warning("How dare someone not of shadow kind, try to communicate with shadows!."))
+		return FALSE
+	if(!((sect.light_power <= -6 - 5 * sect.grand_ritual_level) || (sect.light_reach >= 8 + 7.5 * sect.grand_ritual_level)))
+		to_chat(user, span_warning("You need to strengthen the shadows before you can begin the ritual. Expand shadows to their limits."))
+		return FALSE
+	if(sect.active_obelisk_number < 5 * (sect.grand_ritual_level + 1))
+		to_chat(user, span_warning("You need to anchor the shadows to this reality. You need [5 * (sect.grand_ritual_level + 1)] active obelisks."))
+		return FALSE
+	if(!can_afford(user))
+		return FALSE
+	var/turf/T = get_turf(religious_tool)
+	if(!T.is_holy())
+		to_chat(user, span_warning("The altar can only function in a holy area!"))
+		return FALSE
+	if(!GLOB.religious_sect.altar_anchored)
+		to_chat(user, span_warning("The altar must be secured to the floor if you wish to perform the rite!"))
+		return FALSE
+	spawn()
+		handle_obelisks()
+	return ..()
+
+/datum/religion_rites/grand_ritual_one/invoke_effect(mob/living/user, atom/religious_tool)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	if(sect.active_obelisk_number < 5 * (sect.grand_ritual_level + 1))
+		to_chat(user, span_warning("Your obelisk have been destroyed, destabilizing the ritual!. You need to gather your strength and try again."))
```

Comment:
```suggestion
		to_chat(user, span_warning("Your obelisks have been destroyed, destabilizing the ritual! You need to gather your strength and try again."))
```

Reply:
This is something that only shows up during the end of a ritual which has failed because the final check for obelisks failed. Every major ritual requires multiple obelisks.